README.md

```diff
@@ -1,19 +1,22 @@
 # Amazon SageMaker Debugger
 
+## Table of Contents
+
 - [Overview](#overview)
-- [Examples](#examples)
+- [Support](#support)
 - [How It Works](#how-it-works)
 - [Docs](#docs)
+- [Examples](#examples)
 - [SageMaker Debugger in action](#sagemaker-debugger-in-action)
 
-
 ## Overview
-Amazon SageMaker Debugger is an offering from AWS which help you automate the debugging of machine learning training jobs.
-This library powers Amazon SageMaker Debugger, and helps you develop better, faster and cheaper models by catching common errors quickly.
-It allows you to save tensors from training jobs and makes these tensors available for analysis, all through a flexible and powerful API.
-It supports TensorFlow, PyTorch, MXNet, and XGBoost on Python 3.6+.
+[Amazon SageMaker Debugger](https://docs.aws.amazon.com/sagemaker/latest/dg/train-debugger.html) automates the debugging process of machine learning training jobs. It allows you to save tensors from training jobs and makes these tensors available for analysis, all through a flexible and powerful API.
+`smdebug` library powers [Amazon SageMaker Debugger](https://docs.aws.amazon.com/sagemaker/latest/dg/train-debugger.html) by calling the saved tensors in S3 during the training job. `smdebug` retrieves and filters tensors such as gradients, weights, and biases.
+
+Amazon SageMaker Debugger supports TensorFlow, PyTorch, MXNet, and XGBoost frameworks.
+The following list is a summary of the main functionalities of Amazon SageMaker Debugger:
 
-- Zero Script Change experience on SageMaker when using [supported containers](docs/sagemaker.md#zero-script-change)
+- Zero Script Change experience on SageMaker when using [supported containers](#support)
```

Comment:
There is no mention on ZCC till now in the doc. This is first time user encountered ZCC in this Readme, may be we should provide link to section that explains zcc.